earth_enterprise/src/fusion/mtjtools/scroll/application.cpp

```diff
@@ -105,7 +105,7 @@ void ApplicationWindow::loadInitImage(const std::string &image_file_path) {
 void ApplicationWindow::choose()
 {
   QFileDialog *fdlg = new QFileDialog(prevDir, QString::null, this);
-  fdlg->addFilter("Images (*.pre *.img *.tif *.jp2 *.sid *.jpg *.IMG *.TIF *.JP2 *.SID *.JPG)");
+  fdlg->addFilter("Images (*.pre *.img *.tif *.tiff *.jp2 *.sid *.jpg *.IMG *.TIF *.TIFF *.JP2 *.SID *.JPG)");
```

Comment:
Also please take a look at the documentation and update any text that list those extensions so the doc is in sync with the software change.